pandas/core/arrays/datetimes.py

```diff
@@ -1747,7 +1747,8 @@ def _generate_regular_range(cls, start, end, periods, freq):
     return data
 
 
-def _generate_range_overflow_safe(endpoint, periods, stride, side='start'):
+def _generate_range_overflow_safe(endpoint, periods, stride,
+                                  side='start'):
     """
     Calculate the second endpoint for passing to np.arange, checking
     to avoid an integer overflow.  Catch OverflowError and re-raise
```

Comment:
can you doc-string this a bit more (say what the parameters mean)

Reply:
sure